test/integration/scheduler/filters/filters_test.go

```diff
@@ -1790,3 +1791,230 @@ func TestUnschedulablePodBecomesSchedulable(t *testing.T) {
 		})
 	}
 }
+
+func TestPvTopology(t *testing.T) {
```

Comment:
@msau42 can you give a pass to this?

Reply:
Also I would prefer that this PR #116634 merges first

Reply:
FYI, the test cases in this test are **not** changed. I just moved them from unit tests to integration test.

Reply:
Seems like there will be some conflicts afer #116634 , I think I'll need to rebase again😂, and you can wait to review after that.